packages/fetch/package.json

```diff
@@ -0,0 +1,57 @@
+{
+	"name": "@sapphire/fetch",
+	"version": "0.0.1",
+	"description": "Tiny wrapper around node-fetch for improved TypeScript and data type support",
+	"author": "@sapphire",
+	"license": "MIT",
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
+	"types": "dist/index.d.ts",
+	"exports": {
+		"import": "./dist/index.mjs",
+		"require": "./dist/index.js"
+	},
+	"homepage": "https://github.com/sapphire-community/utilities/tree/main/packages/fetch",
+	"scripts": {
+		"test": "jest",
+		"lint": "eslint src tests --ext ts --fix -c ../../.eslintrc",
+		"build": "tsc -b src",
+		"postbuild": "gen-esm-wrapper dist/index.js dist/index.mjs",
+		"start": "yarn build -w",
+		"prepublishOnly": "yarn build"
+	},
+	"dependencies": {
+		"@types/node-fetch": "^2.5.10",
```

Comment:
Dev dep, skipLibCheck on user side imo. @kyranet thoughts?

Reply:
EW NO! `skipLibCheck` is horrid.

Reply:
Did I stutter?